django/db/backends/sqlite3/schema.py

```diff
@@ -47,6 +47,12 @@ def _remake_table(self, model, create_fields=[], delete_fields=[], alter_fields=
                       override_indexes=None):
         """
         Shortcut to transform a model from old_model into new_model
+
```

Comment:
This change seems perhaps worthwhile, but why is it in the same PR as the app-registry stuff? It seems (at least mostly) unrelated.